docs/monitoring/monitoring.md

```diff
@@ -16,3 +16,59 @@ Also, users can click the [Grafana](http://grafana.com/) dashboard link to view
 ## VM Detail Metrics
 For each VM, users can view the VM metrics by clicking the VM details page.
 ![](./assets/vm-metrics.png)
+
+
+## Change resources setting of Monitoring
+_Available as of v1.0.1_
+
+Monitoring has a couple of components to collect metrics data from all NODEs/PODs/VMs/... and summarize them. The resources required by Monitoring are related to workloads and hardware resources. Harvester sets the default values according to engineering practices, users can change them accordingly.
```

Comment:
Is there any reference for the `engineering practices` mentioned?

Reply:
I do not have an reference data/article, it is more from an roughly estimation based on daily work and experience.